lib/rocket_pants/controller/respondable.rb

```diff
@@ -5,14 +5,17 @@ module Respondable
     SerializerWrapper = Struct.new(:serializer, :object) do
 
       def serializable_hash(options = {})
```

Comment:
coming here from: https://github.com/rails-api/active_model_serializers/issues/1321#issuecomment-165070164

could be a little more efficient:

if ActiveModel::Serializer::VERSION.start_with?('0.9.')
        def serializable_hash(options = {})
          instance = serializer.new(object, options)
          if instance.respond_to?(:serializable_hash)
            instance.serializable_hash
          else
            instance.as_json options
          end
        end
elsif defined?(ActiveModel::SerializableResource)
  def serializable_hash(options = {})
    ActiveModel::SerializableResource.new(object, options.merge(serializer: serializer)).serializable_hash
  end
else
  fail ArgumentError, "We don't know how to serialize in #{ActiveModel::Serializer::VERSION}"
end